scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -685,7 +687,7 @@ class Framework extends SbtFramework {
     val threadFactory =
       new ThreadFactory {
         val defaultThreadFactory = Executors.defaultThreadFactory
-        def newThread(runnable: Runnable): Thread = {
+        def newThread(runnable: Runnable | Null): Thread = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.